Source/Particles/Collision/CollisionHandler.cpp

```diff
@@ -68,6 +69,12 @@ CollisionHandler::CollisionHandler(MultiParticleContainer const * const mypc)
                     collision_names[i], mypc
                 );
         }
+        else if (type == "photonphoton") {
+            allcollisions[i] =
+               std::make_unique<BinaryCollision<PhotonPhotonCollisionFunc, ParticleCreationFunc>>(
+                    collision_names[i], mypc
+               );
+        }
```

Comment:
Looking at this file, it seems that we allow these input strings:
- `pairwisecoulomb`
- `background_mcc`
- `background_stopping`
- `dsmc`
- `nuclearfusion`
- `photonphoton`

I think that concatenating two or more words without some kind of formatting (like, `snake_case`, `camelCase` or `PascalCase`, etc.) is not super readable, especially for user-facing inputs.

While I know that renaming `pairwisecoulomb` and `nuclearfusion` would be beyond the scope of this PR, I'm wondering if we could rename `photonphoton` to `photon_photon`, following the same style as `background_mcc` and `background_stopping`. 

Just a personal preference, so up to you/other reviewers.